www/hoan01/sem-1/styles.css

```diff
@@ -0,0 +1,243 @@
+}
+
+h1 {
+	font-size: 50px;
+	font-family: 'Lobster', sans-serif;
+}
+
+h1,h2{
+	color: white;
+	text-shadow: 15px 10px 20px black;
+}
+
+h3{
+	font-size: 40px;
+}
+
+div button:hover {
+  cursor: pointer;
+}
+
+audio {
+	position: fixed;
+	right:0px;
+	top:0px;
+}
+/* ---End styling all html docs--->
+
+/* ---Styling Home Page--- */
+.main button {
+  width: 50%;
+  height: 90%;
+  font-size: 30px;
+  font-family: Times New Roman;
+  padding: 30px;
+  margin-bottom: 5px;
+  border-radius: 22px;
+  background-color: white;
+}
+
+.header {
+	background-image: url();
+}
+
+.main {
+	background: url('https://www.kidschat.net/images/left-picture.png');
+}
+
+.footer {
+	padding: 22px;
+	background-color: black;
+	color:white;
+}
+
+/* ---End styling Home Page--- */
+
+/* ---Styling Alphabet--- */
+.content-0 div{
+	display: block;
+	font-size: 22px;
+	text-align: center;
+	border: 1px solid black;
+	width:50%;
+}
+
+.content-0 div:nth-child(even){
+	float:left;
+}
+
+.content-0 div:nth-child(odd){
+	float:right;
+}
+
+.content-0 div:nth-child(odd)::first-letter{
+	text-decoration: underline;
+	color: red;
+}
+
+.doubleLetter {
+	text-decoration: underline;
+	color: red;
+}
+
+.txt-2 {
+	width: 1em;
+	height: 1em;
+	font-size: 22px;
+	text-align: center;
+}
+
+/* ---End styling Alphabet--- */
+
+/* ---Styling Colors--- */
+
+.content-1 div {
+	border: 2px dotted black;
+	padding: 80px;
+	font-size: 22px;
+}
+
+.colGreen {
+	background-color: rgb(0,255,0);
+	float: left;
+	width: 66.6666%;
+	height: 200px;
+}
+
+.colBlue {
+	background-color: rgb(0,0,255);
+	float: right;
+	width: 33.3333%;
+	height: 200px;
+}
+
+.colRed {
+	background-color: rgb(255,0,0);
+	float: left;
+	width: 33.3333%;
+	height: 200px;
+}
+
+.colYellow {
+	background-color: yellow;
+	float: right;
+	width: 66.6666%;
+	height: 200px;
+}
+
+.colWhite {
+	background-color: rgb(255,255,255);
+	float: left;
+	width: 50%;
+	height: 200px;
+}
+
+.colBlack {
+	background-color: rgb(0,0,0);
+	float: right;
+	width: 50%;
+	height: 200px;
+}
+
+.txt {
+	height: 2em;
+	padding-top: 10px;
+	font-size: 22px;
+	text-align: center;
+}
+
+/* ---End styling Colors--- */
+
+
+/* ---Styling Counting--- */
+
+.content-2::after {
+	content: "";
+    display: table;
+    clear: both;
+}
+
+.content-2 div {
+	float: left;
+    width: 33.33%;
+    font-size: 22px;
+    padding-bottom: 5px;
+}
+
+.num {
+	width: 2em;
+	height: 2em;
+	padding-top: 10px;
+	font-size: 22px;
+	text-align: center;
+}
+
+input[type=number]::-webkit-inner-spin-button, 
+input[type=number]::-webkit-outer-spin-button { 
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    margin: 0; 
+}
+
+/* ---End styling Counting--- */
+
+/* ---Styling English--- */
+
+
+.content-3 div {
+	float: left;
+    width: 50%;
+    font-size: 28px;
+    border: 1px solid black;
+}
+
+.txt-1 {
+	font-size: 25px;
+	text-align: center;
+}
+
+/* ---End styling English--- */
+
+/* ---Responsible web---*/
```

Comment:
tak zodpovedny web to slysim poprve, ani tak mi to pripada jako moc velka zodpovednost